editor/import/resource_importer_texture.cpp

```diff
@@ -533,8 +582,10 @@ Error ResourceImporterTexture::import(ResourceUID::ID p_source_id, const String
 
 		editor_image.instantiate();
 		err = ImageLoader::load_image(p_source_file, editor_image, nullptr, editor_loader_flags, editor_scale);
+
 		if (err != OK) {
-			WARN_PRINT("Failed to import an image resource for editor use from '" + p_source_file + "'");
+			WARN_PRINT(vformat("Failed to import an image resource for editor use from '%s'.", p_source_file));
+
```

Comment:
I understand that adding such whitespace might make it marginally easier to parse, but our current convention is to go away from that old habit which we can't reasonably have done consistently.

Especially in such a case with a one liner and short `if` and no `else` condition, I think it's actually better without.

There are other places where you added such empty lines before `} else {` in this PR and in #98100 which arguably shouldn't have been done IMO, but it's not a big deal so I wouldn't block merge for this.